### DISCUSSION TITLE
How to manually reconcile?

### DISCUSSION THREAD

**Discussion by nayab9** [layday/instawow#107] [Q&A]
I have an addon installed (Dominos) - this addon is not picked up by the reconcile command.

Since this addon seems to have releases on github ( https://github.com/tullamods/Dominos ); what is the best way to update instawow so it points to github?

Do I open a PR against https://github.com/layday/github-wow-addon-catalogue/blob/main/addons.csv, and add the URL there?

Wondering if some sort of local command would be useful, like a

instawow reconcile addon-name url

To update my local version, so it can be migrated, at least until it is added to the addons.csv file?

Alternatively, I could tell instawow to remove my current install, then install it via the github URL - but would this potentially blow away my saved data for that addon when I uninstall it? Or are they stored separately so I wouldn't lose anything?

**Comment by layday** [comment]
> Do I open a PR against https://github.com/layday/github-wow-addon-catalogue/blob/main/addons.csv, and add the URL there?

Dominos is in the catalogue but its Curse ID is not listed in the add-on's TOC file so we're not able to extract it automatically.  There would need to be a way to augment collected add-on metadata, which currently there isn't.

> Alternatively, I could tell instawow to remove my current install, then install it via the github URL - but would this potentially blow away my saved data for that addon when I uninstall it? Or are they stored separately so I wouldn't lose anything?

The add-on data are stored separately.  Uninstalling an add-on never deletes its data.

The easiest solution to your problem is:

```
instawow remove curse:dominos && instawow install github:tullamods/dominos
```

Or:

```
instawow search dominos
```

to see if it's available from other sources.

**Reply by nayab9** [reply]
fantastic, good to know the data will be saved even if uninstalling the addon!

Both these work like a charm, just tried them on 2 separate addons.

Thanks so much

**Reply by layday** [reply]
No problem :)
